app/models/post.js

```diff
@@ -74,16 +79,14 @@ exports.addModel = function(database) {
 
       that.validateOnCreate()
         .then(function(post) {
-          return Promise.all([
-            database.hmsetAsync(mkKey(['post', post.id]),
-                                { 'body': post.body,
-                                  'userId': post.userId,
-                                  'createdAt': post.createdAt.toString(),
-                                  'updatedAt': post.updatedAt.toString()
-                                }),
-            models.Timeline.newPost(post.id, that.timelineIds)
```

Comment:
Created a story to review #create methods https://github.com/pepyatka/pepyatka-server/issues/131. Good catch.